src/ui/units/dash/components/TwoColumnDialog/TwoColumnDialog.scss

```diff
@@ -22,9 +22,12 @@
 
     &__content {
         overflow-y: auto;
+        width: 705px;
 
-        &:not(&_no-sidebar) {
-            width: 705px;
+        &_no-content-header {
+            // TODO: this is an private api and may silently changed. We should use local styles
+            // Used values from redefined --yc-dialog-header-padding
+            padding-top: calc(var(--g-text-subheader-3-line-height) + 30px + 16px);
```

Comment:
Let's move `30px` and `16px` to local css constants with good naming (to understand quickly which item is what for)